usaspending_api/tests/conftest_spark.py

```diff
@@ -48,9 +59,10 @@
 
 
 @fixture(scope="session")
-def s3_unittest_data_bucket_setup_and_teardown():
+def s3_unittest_data_bucket_setup_and_teardown(worker_id):
```

Comment:
You should duck type this new parameter and describe it in the pydoc.

Reply:
Done